### ISSUE TITLE
CSS conflict with deck.js

### ISSUE THREAD

**Issue by viebel** [codemirror/CodeMirror#4193]
In a [deck.js](http://imakewebthings.com/deck.js/) presentation codemirror elements behave weirdly.

It's probably a CSS conflict. Please advice.

Here is an example with a 2-slides presentation that illustrate the issues:

https://viebel.github.io/deckjs-demo-klipse/codemirror/index.html

I have 2 issues:

1. selection of part of the code is not highlighted
2. code navigation with keyboard arrows is broken

**Comment by marijnh** [comment]
Indeed, probably a CSS conflict. I couldn't figure out why or how though.
